frame/support/src/storage/bounded_vec.rs

```diff
@@ -122,6 +122,14 @@ impl<T, S> BoundedVec<T, S> {
 	pub fn retain<F: FnMut(&T) -> bool>(&mut self, f: F) {
 		self.0.retain(f)
 	}
+
+	/// Exactly the same semantics as [`Vec::get_mut`].
+	pub fn get_mut<I: SliceIndex<[T]>>(
```

Comment:
👍

Reply:
There should also be the same addition to the `weakly_bounded_vec` definition